cli.ts

```diff
@@ -66,6 +68,23 @@ async function cli(args: ParsedArgs) {
     validateStatus: () => true,
   })
 
+  if ("action_attempt" in response.data) {
+    const { poll_for_action_attempt } = await prompts({
+      name: "poll_for_action_attempt",
+      message: "Would you like to poll the action attempt until it's ready?",
```

Comment:
Should we also add something like `This request returns an action attempt. Would you like to poll the action attempt until it's ready?`?

Reply:
I think what you have is ok!